buildscripts/incremental/setup_conda_environment.sh

```diff
@@ -83,6 +83,9 @@ fi
 # Install latest llvmlite build
 $CONDA_INSTALL -c numba/label/dev llvmlite
 
+# Install latest ipykernel for testing
+$CONDA_INSTALL ipykernel ipython_genutils
```

Comment:
This will interfere with the deliberately empty setup created by the `VANILLA_INSTALL` condition/env var. Suggest adding `ipython_genutils` to this line instead:
https://github.com/numba/numba/blob/ad5dec94dbf61c5b1e171f237bb0d9ac788fcfd2/buildscripts/incremental/setup_conda_environment.sh#L58

Reply:
Sounds good, done.